doctrine-phpcr-odm-mapping.xsd

```diff
@@ -44,6 +44,15 @@
         <xs:anyAttribute namespace="##other"/>
     </xs:complexType>
 
+    <xs:complexType name="child-class">
+        <xs:simpleContent>
+            <xs:extension base="xs:string">
+                <xs:attribute name="value" type="xs:string">
+                </xs:attribute>
```

Comment:
@WouterJ what is best practice here? using an element with an attribute or make the class the text content of the xml tag?